ee_tests/src/specs/homepage.spec.ts

```diff
@@ -34,10 +34,11 @@ describe('HomePage', () => {
     ]);
   });
 
-  it('can login using correct username and password', async () => {
+  it('can login using a valid username and password', async () => {
     support.debug('... starting test; loginPage');
     let loginPage = await homePage.gotoLoginPage();
     support.debug('... back from gotoLoginPage');
-    await loginPage.login(browser.params.login.user, browser.params.login.password);
+    let dashboardPage = await loginPage.login(browser.params.login.user, browser.params.login.password);
+    dashboardPage.validate();    
```

Comment:
we don't have to validate the object since the `loginPage.login` must return only valid object; i.e. it (login) must call `await dashboardPage.validate()`. Please also note that we need to add `await` in front of `dashboardPage.validate()` as `login` is an `async` method and returns a promise.

Reply:
Great suggestion on lint - I actually just installed it into xcode.